5-AccessControl/2-call-api-groups/API/auth/guard.js

```diff
@@ -0,0 +1,24 @@
+const handleOverage = require('./overage');
+const { requestHasRequiredAttributes } = require('../utils/permissionUtils');
+
+const routeGuard = (accessMatrix) => {
+    return async (req, res, next) => {
+        if (req.authInfo.groups === undefined) {
+            if (req.authInfo['_claim_names'] || req.authInfo['_claim_sources']) {
```

Comment:
The exact check should be: 

if (req.authInfo.hasOwnProperty("_claim_names") && req.authInfo["_claim_names"].hasOwnProperty("groups"))

Reply:
I don't think this is resolved